lib/index.d.ts

```diff
@@ -184,24 +174,24 @@ type EncryptOptionsPmcrypto = EncryptOptionsPmcryptoWithData | EncryptOptions;
 
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor?: true; detached?: false }
-): Promise<EncryptResult<string>>;
+): Promise<EncryptResult<undefined, string>>;
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor?: true; detached: true }
-): Promise<EncryptResult<string, undefined, string, string>>;
+): Promise<EncryptResult<undefined, string, Uint8Array, string>>;
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor: false; detached?: false }
-): Promise<EncryptResult<undefined, message.Message>>;
+): Promise<EncryptResult<undefined, OpenPGPMessage>>;
 export function encryptMessage(
     options: EncryptOptionsPmcrypto & { armor: false; detached: true }
-): Promise<EncryptResult<undefined, message.Message, OpenPGPSignature, message.Message>>;
+): Promise<EncryptResult<undefined, OpenPGPMessage, Uint8Array, OpenPGPMessage>>;
```

Comment:
Do we want the change in output types?

Reply:
Maybe, if we anyway want to change from `armor` to `format` parameter, then it's reasonable to change the output type too, yeah?

Reply:
Yes but not in this way